clients/src/main/java/org/apache/kafka/common/Cluster.java

```diff
@@ -45,14 +46,16 @@
      */
     public Cluster(Collection<Node> nodes,
                    Collection<PartitionInfo> partitions,
-                   Set<String> unauthorizedTopics) {
-        this(false, nodes, partitions, unauthorizedTopics);
+                   Set<String> unauthorizedTopics,
+                   Set<String> internalTopics) {
```

Comment:
Is there any need to maintain the old constructor here? I don't want to create telescoping constructors either though. At some point if we keep adding things to Cluster a builder may be appropriate.